stdlib/public/core/Integers.swift.gyb

```diff
@@ -1128,13 +2614,17 @@ extension ${Self} {
 //                  FixedWidthInteger.init(_truncatingBits:)
 %   if should_define_truncating_bit_pattern_init(src_ty=src_type, dst_ty=self_type):
 extension ${Self} {
-  /// Construct a `${Self}` having the same bitwise representation as
-  /// the least significant bits of the provided bit pattern.
+  /// Creates a new instance with the same bitwise representation as the least
+  /// significant bits of the given value.
+  ///
+  /// This initializer performs no range or overflow checking. The resulting
+  /// instance may have a different numeric value from `source`.
   ///
-  /// No range or overflow checking occurs.
+  /// - Parameter source: An integer to use as the source of the new value's
+  ///   bit pattern.
   @_transparent
-  public init(truncatingBitPattern: ${Src}) {
-    let src = truncatingBitPattern._value
+  public init(truncatingBitPattern source: ${Src}) {
```

Comment:
I should have said `init(truncatingBitPattern:)` is subsumed by `init(extendingOrTruncating:)`—as far as I can tell,  the two have the same behavior:
```swift
let x: UInt16 = 4044                  // 0b00001111_11001100

UInt8(extendingOrTruncating: x)       // 204 == 0b11001100
UInt8(truncatingBitPattern: x)        // 204 == 0b11001100

Int8(extendingOrTruncating: x)        // -52 == 0b11001100
Int8(truncatingBitPattern: x)         // -52 == 0b11001100
```
If I'm missing something I'll need to revisit the docs!